qmlti/PerceptionSoap.php

```diff
@@ -250,10 +291,12 @@ public function get_access_assessment_notify($assessment_id, $participant_name,
         "Notify" => $notify_url,
         "ParameterList" => array(
           "Parameter" => array(
-            array("Name" => "home", "Value" => $home_url),
+            array("Name" => "HOME", "Value" => $home_url),
             array("Name" => "lti_consumer_key", "Value" => $consumer_key),
             array("Name" => "lti_context_id", "Value" => $resource_link_id),
             array("Name" => "lti_result_id", "Value" => $result_id),
+            array("Name" => "coaching_report", "Value" => $coaching_report),
```

Comment:
shouldn't you make this optional for this method so that consumers have the option?